main.py

```diff
@@ -86,19 +90,31 @@ def extract_transform_and_load_covid_data(post_to_google_sheets=True):
     #     credentials=credentials,
     # )
 
-    covidtracking_df = extract_covidtracking_historical_data()
-    cdc_ili_df = extract_cdc_ili_data()
+    covidatlas_df = extract_covid_atlas_data()
```

Comment:
If we don't think we'll ever need to mix county-level and state-level data, I'm wondering if we want to actually put this in a separate function -- and maybe the function is invoked separately by `main()` so it's easy to run one command that updates all data, but there's a bit cleaner separation of concerns. 

I feel like `extract_transform_and_load_covid_data` has gotten to be a beast and now might be a good time to refactor a bit!

Reply:
agreed it would be better, but the status quo here isn't much different. We don't do a lot where we merge all the state data into one state object and then do additional things with that. As far as `main` is concerned, county-data is similar to state-level beds data. Happy to refactor, but perhaps we could save some of these for some "internal code cleanup" tasks once the county-level functionality is squared away